cogs/check.py

```diff
@@ -33,40 +37,44 @@ def __init__(self, bot):
 
         await channel_setup(ctx)
         await user_setup(ctx)
-
-        global achievement
```

Comment:
Will achievement still be accessible locally if this line is removed?

Reply:
global statements are only needed if you're assigning to a global value(which you're not)

Reply:
Ah, ok!